src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/RemoveResourceDesignerStep.cs

```diff
@@ -7,19 +7,33 @@
 using System.Collections.Generic;
 using Mono.Cecil.Cil;
 using System.Text.RegularExpressions;
+#if ILLINK
+using Microsoft.Android.Sdk.ILLink;
+#endif
 
 namespace MonoDroid.Tuner
 {
-	public class RemoveResourceDesignerStep : BaseStep
+	public class RemoveResourceDesignerStep : LinkDesignerBase
 	{
 		TypeDefinition mainDesigner = null;
 		AssemblyDefinition mainAssembly = null;
 		CustomAttribute mainDesignerAttribute;
 		Dictionary<string, int> designerConstants;
 		Regex opCodeRegex = new Regex (@"([\w]+): ([\w]+) ([\w.]+) ([\w:./]+)");
-
+#if ILLINK
 		protected override void Process ()
 		{
+			cache = Context;
+		}
+#else
+		readonly
+#endif
+		IMetadataResolver cache;
```

Comment:
I was looking through here, is this `cache` variable unused?

Reply:
I'll try removing it seem if it impacts anything.